src/EditorFeatures/CSharpTest/Diagnostics/GenerateMethod/GenerateMethodTests.cs

```diff
@@ -2919,6 +2919,83 @@ private static OtherClass InitializeProperty()
 }");
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateMethod)]
+        [WorkItem(12708, "https://github.com/dotnet/roslyn/issues/12708")]
+        public async Task TestNoOverloadOnEventAssignment()
+        {
+            await TestAsync(
+@"class Program
+{
+    void Main(string[] args)
+    {
+        [|myEvent += OnChanged|];
+    }
+    delegate void MyDel(int arg);
+    static event MyDel myEvent;
+
+    private void OnChanged() { }
+}
+",
+@"using System;
+
+class Program
+{
+    void Main(string[] args)
+    {
+        myEvent += OnChanged;
+    }
+
+    private void OnChanged(int arg)
+    {
+        throw new NotImplementedException();
+    }
+
+    delegate void MyDel(int arg);
+    static event MyDel myEvent;
+
+    private void OnChanged() { }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateMethod)]
+        [WorkItem(12708, "https://github.com/dotnet/roslyn/issues/12708")]
+        public async Task TestNoOverloadOnEventAssignmentBracketed_Bracketed()
```

Comment:
`Bracketed_Bracketed`:

1. We tend to be pendantic, and use `bracket` to mean `[` or `]`.  `(` and `)` are parentheses.
2. Not sure what you are trying to convey with the underscore break.